docs/index.rst

```diff
@@ -19,11 +19,13 @@ Catkit2
    :maxdepth: 1
    :caption: Built-In Services
 
-   service/aimtti_plp
+   services/aimtti_plp
+   services/allied_vision_camera
    services/bmc_dm
```

Comment:
I don't know if we have to add them, but every simulated service is not added, even though the camera_sim is included in the list

Reply:
Also true! We were initially not entirely sure whether we should include docs for the the simulated services here or not. Since a simulated service is supposed to have exactly the same behavior like a hardware service, minus all the setup that is required for a hardware service, I recently deleted all docs for simulated services in this list.

The reason the documentation for the simulated service `camera_sim` is still there because that one is somewhat special: it is a very general simulated service that can be used as the simulated service for any camera. So it seemed a good idea to give it its own doc page.

I admit these choices are somewhat arbitrary, but they are leaning on how we are currently using catkit2 and all its services. This might well be adjusted in the future

Reply:
ok perfect !